src/shared/components/noRecords/noRecords.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import { Box, Flex, Text } from "@mantine/core";
+import { IconMoodSadDizzy } from "@tabler/icons-react";
+
+const NoRecords: FC = () => {
+  return (
+    <Flex
+      direction={"column"}
+      align={"center"}
+      sx={{ height: "75vh" }}
+      justify={"center"}
+    >
+      <Box
+        p={4}
+        mb={4}
+        sx={(theme) => ({
+          width: "48px",
+          fontSize: 0,
+          color:
+            theme.colorScheme === "dark"
```

Comment:
there is no need of dark theme condition

Reply:
why?

Reply:
do you use dark theme in this application?

Reply:
no

Reply:
done